app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1602,44 +1602,53 @@ table li .account_details {
   margin-bottom: 50px;
 }
 
-.row-items {
-  max-width: 1000px;
+// start non-admin items menu
+.items table {
+  background-color: #202020;
```

Comment:
Color literals like `#202020` should only be used in variable declarations; they should be referred to via variable everywhere else.